packages/@sfcc-core/core-graphql/src/graphql.ts

```diff
@@ -139,6 +139,14 @@ export class CoreGraphQL {
             this.apolloServer.applyMiddleware({
                 app: expressApp,
                 path: apiPath,
+                cors: apiConfig.COMMERCE_CORS
```

Comment:
By default, right now CORS is enabled. Turn it off unless an environment variable is set.

Reply:
Can we please have some documentation around it?